.setup/CONFIGURE_SUBMITTY.py

```diff
@@ -494,6 +494,17 @@ def write(x=''):
 
 IGNORED_FILES_AND_DIRS = ['saml', 'login.md']
 
+system_message = None
+# this prevents clobbered system_message
+try:
+    with open(SUBMITTY_JSON,'r') as submitty_config:
+        load_submitty_json = json.load(submitty_config)
+        if 'system_message' in load_submitty_json:
```

Comment:
Here we're giving special importance to the 'system_message' key.
What we can do instead is use the existing JSON as the base for the new data, which means any key that we don't change will not get deleted.

Reply:
I agree that in general we want to keep everything, but if we take a look at Config.php where these json tags are parsed and used, system_message is the only significant key that needs special importance.
I think that is what the initial issue expects to address. Hence, the request is beyond the scope of original issue. 
You could make it an additional issue and mention this PR instead.

Reply:
(Discussed in meeting, @bmcutler requested for all keys to be preserved. PR scope was updated to meet this.)